### ISSUE TITLE
content loss calculation is wrong?

### ISSUE THREAD

**Issue by RyanCV** [abhiskk/fast-neural-style#24]
Hi, I found the vgg16 loss network returns [relu1_2, relu2_2, relu3_3, relu4_3], and in your neural_style.py, 
```
y = transformer(x)
features_y = vgg(y)
f_xc_c = Variable(features_xc[1].data, requires_grad=False)
content_loss = args.content_weight * mse_loss(features_y[1], f_xc_c)
```
the features_y is a list of loss corresponding to  [relu1_2, relu2_2, relu3_3, relu4_3], the index for relu1_2, relu2_2, relu3_3 and relu4_3 is 0, 1, 2, and 3, right? In the paper, it used relu3_3 as content loss, so, here should it be the following?
`f_xc_c = Variable(features_xc[2].data, requires_grad=False)
content_loss = args.content_weight * mse_loss(features_y[2], f_xc_c)`
Thanks.

**Comment by abhiskk** [comment]
From the paper [Johnson et al.](https://arxiv.org/pdf/1603.08155.pdf), Training details on page 9:
```
For all style transfer experiments we compute
feature reconstruction loss at layer relu2_2
```
